Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -334,20 +335,21 @@ private void Esc()
         public Settings Settings { get; }
         public string ClockText { get; private set; }
         public string DateText { get; private set; }
+        public CultureInfo Culture => InternationalizationManager.Instance.CurrentCulture;
 
-        public CultureInfo cultureInfo => new CultureInfo(Settings.Language);
         private async Task RegisterClockAndDateUpdateAsync()
         {
             var timer = new PeriodicTimer(TimeSpan.FromSeconds(1));
             // ReSharper disable once MethodSupportsCancellation
             while (await timer.WaitForNextTickAsync().ConfigureAwait(false))
             {
                 if (Settings.UseClock)
-                    ClockText = DateTime.Now.ToString(Settings.TimeFormat, cultureInfo);
+                    ClockText = DateTime.Now.ToString(Settings.TimeFormat, Culture);
```

Comment:
I am confused why this is needed. Isn't `CultureInfo.CurrentCulture` is changed whenever language is changed?

Reply:
CultureInfo.CurrentCulture is per thread.

Reply:
Updated, plz check again.